test/test.options.bool.js

```diff
@@ -1,15 +1,28 @@
-/**
- * Module dependencies.
- */
+const commander = require('../');
+// eslint-disable-next-line no-unused-vars
+const should = require('should');
 
-var program = require('../')
-  , should = require('should');
+// Test simple flag and negatable flag
 
-program
-  .version('0.0.1')
-  .option('-p, --pepper', 'add pepper')
-  .option('-c, --no-cheese', 'remove cheese');
+function simpleFlagProgram() {
+  const program = new commander.Command();
+  program
+    .option('-p, --pepper', 'add pepper')
+    .option('-C, --no-cheese', 'remove cheese');
+  return program;
+}
 
-program.parse(['node', 'test', '--pepper']);
-program.pepper.should.be.true();
-program.cheese.should.be.true();
+const simpleFlagNoOptions = simpleFlagProgram();
+simpleFlagNoOptions.parse(['node', 'test']);
+simpleFlagNoOptions.should.not.have.property('pepper');
+simpleFlagNoOptions.cheese.should.be.true();
+
+const simpleFlagLong = simpleFlagProgram();
+simpleFlagLong.parse(['node', 'test', '--pepper', '--no-cheese']);
+simpleFlagLong.pepper.should.be.true();
+simpleFlagLong.cheese.should.be.false();
+
+const simpleFlagShort = simpleFlagProgram();
```

Comment:
This test seems a little redundant here since it's really testing flags, but fine to keep it.

Reply:
Actually I deleted the separate file that was testing boolean short flags, but I am regretting that. I'll put the file back and remove this explicit test!

Reply:
Restored. Done.